### PR TITLE
Change Json response format and update sample

### PR DESCRIPTION
**Pull request by da-shino**
Update attendances JSON sample

### CODE REVIEW

File: test/response/api/v1/attendances/get.json [hunk 1]
```diff
@@ -1,337 +1,504 @@
...
+        "id": 3,
+        "href": "http://amam.sample.com/api/v1/teams/3"
+      },
+      "member": {
+        "id": "100606116",
+        "href": "http://amam.sample.com/api/v1/members/100606116"
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "http://amam.sample.com/api/v1/attendance_types/5"
```
Comment (ninja-inc) [inline test/response/api/v1/attendances/get.json:15]: [Underscores vs. hyphens in URLs](http://ux.stackexchange.com/questions/9303/underscores-vs-hyphens-in-urls)
[Google](https://support.google.com/webmasters/answer/76329) recommended to use hyphenated URLs.

But for famous public APIs (Facebook, Netflix, etc..) seem using underscored URLs.
[Hyphen, underscore, or camelCase as word delimiter in URIs?](http://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris)

What do you think? @sideroad @nabuchi @taka66

File: test/response/api/v1/attendances/get.json [hunk 2]
```diff
@@ -1,337 +1,504 @@
...
+      },
+      "attendanceReason": {
+        "id": 6,
+        "href": "http://amam.sample.com/api/v1/attendance_reasons/6"
+      },
+      "remarks": "中国",
+      "createdAt": "2016-08-25T19:01:19.000+09:00",
+      "updatedAt": null,
+      "date": "2016-12-26",
```
Comment (ninja-inc) [inline test/response/api/v1/attendances/get.json:24]: It depends on UI side but could we show start and end datetime considering time zone? It's hard for us to know accurate arrival time at the office for not Tokyo office members.

Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json:24]: like a `"startDateTime": "2016-08-25T10:00:00.000+09:00"`

**Review by ninja-inc — CHANGES_REQUESTED** [review]

File: test/response/api/v1/attendances/get.json (hunk 2 above)
Comment (da-shino) [reply test/response/api/v1/attendances/get.json:24]: Exactly, it should not depend time zone but we need more consideration and discussion.a
Currently "date" is one of composite unique key of Attendance model and it has index also. So, if changed start/end model, system should be changed significantly.

Not changed key and just changes format with 0 pad like "YYYY-MM-DD T00:00:00.000+09:00" is no problem at this moment I think.

File: test/response/api/v1/attendances/get.json (hunk 1 above)
Comment (sideroad) [reply test/response/api/v1/attendances/get.json:15]: I also like hypen.

Comment (taka66) [reply test/response/api/v1/attendances/get.json:15]: hypen +1

**Comment by da-shino** [comment]
Discard PR due to Architecture change

**Pull request closed without merging**